relayer/processor/metrics.go

```diff
@@ -34,10 +37,15 @@ func (m *PrometheusMetrics) SetFeesSpent(chain, key, address, denom string, amou
 	m.FeesSpent.WithLabelValues(chain, key, address, denom).Set(amount)
 }
 
+func (m *PrometheusMetrics) SetClientExpiration(pathName, chain, clientID, trustingPeriod string, timeToExpiration time.Duration) {
```

Comment:
Okay, it is good to know the value does not change often, but it is still different for different clients leading to many variations. Unless we displayed it differently, maybe with a rounded figure like `428h`, ignoring the minutes and seconds.

Reply:
@nourspace 
For context, the `trustingPeriod` is tied to the client and we are already using the `client-id`(`chain`) as label...
A client can't have multiple trustingPeriods. 
So this is just very handy info to have, especially when troubleshooting why clients have expired.

As for how we display it, I picked decimal/scientific notation format because that is how we are displaying the `cosmos_relayer_client_expiration_seconds` metric. I figured it would be easier to do math between these two metrics if we wanted to.

For example:

If we have a light client that has a trusting period of 1.5408e+06 seconds (428 hours), we could fire an alter if:
`cosmos_relayer_client_expiration_seconds < trustingPeriod/2)`

So if the client gets to a point were it hasn't been updated in 214 hours, we can fire an alert letting us know that the client is half way to expiration. 

With that said, I am not familiar with how these metrics will be ingested. So if displaying `428h` is better, I'll make the change.

Reply:
The alert idea is excellent, but unfortunately, we can't do such operations on the string labels, [afaik](https://stackoverflow.com/questions/75899008/how-can-i-make-arithmetic-operations-division-on-label-values-for-one-metric).

We would have to expose another gauge metric, `cosmos_relayer_trusting_period_seconds` for example. The value of this metric would be float and could be used in arithmetic operations to query remaining time and trigger alerts.

Reply:
It is not necessary to round the trusting_period value. I only thought of having fewer variations should we decide to use this label in dashboards.

For the alert calculations, we would alert on client_expiration < 24h or some other fixed value. That, unless we extract the label into own `cosmos_relayer_trusting_period_seconds` metric. This way we could alert on
`cosmos_relayer_client_expiration_seconds < cosmos_relayer_trusting_period_seconds/2`

Reply:
Copy that. I think having its own metric is the way to go. I will do this in a separate PR since this has already been approved.